### PR TITLE
Fix bug 1215789. Added "copy text" button on the terminology page.

### PR DESCRIPTION
**Pull request by jotes**
Hi @mathjazz, could you review?
Thanks in advance!

### CODE REVIEW

**Comment by mathjazz** [comment]
Thanks for the patch!

The code looks good.

I would change the UI to match the behaviour of the helper tabs in translate view:
* [ ] Remove the button
* [ ] When hovering over search result: change `li` background color to `#3f4752`, set `cursor: pointer` and also display tooltip "Copy to clipboard".
* [ ] You will also need to set `padding-left` and `padding-right` to 5 px.
* [ ] On click, display "Copied!" notification in `li header`, aligned left.

**Comment by jotes** [comment]
'k, no problem.

**Comment by jotes** [comment]
@mathjazz I've updated PR, could you look again?

File: pontoon/base/static/js/main.js [hunk 1]
```diff
@@ -723,6 +723,21 @@ $(function() {
...
+    var successMessage = $('<span class="clipboard-success">Copied!</span>'),
+        $trigger = $(event.trigger);
+
+    $('.clipboard-success').remove();
+    $trigger.find('header').prepend(successMessage);
+    setTimeout(function() {
+      successMessage.fadeOut(500, function() {
+         successMessage.remove();
+      });
+    }, 500);
```
Comment (mathjazz) [inline pontoon/base/static/js/main.js]: My instinct says it feels better if you change this value to 1000. But I have no scientific proof for that. ;)

File: pontoon/base/static/css/style.css [hunk 2]
```diff
@@ -1218,3 +1217,16 @@ body > .container {
   font-style: italic;
   margin-top: 20px;
 }
+
+.search .clipboard-success {
+  float: left;
```
Comment (mathjazz) [inline pontoon/base/static/css/style.css]: Please also set `color: #7BC876;` for more contrast.

**Comment by mathjazz** [comment]
Excellent!

Please move CSS to `search.css`and JS to `search.js`.

Please also add `title="Copy to clipboard"` to the `li` element. You can set this in `main.js`- `getMachinvery()`- `append()`.

File: pontoon/base/static/css/style.css [hunk 3]
```diff
@@ -1128,8 +1128,7 @@ body > .container {
 
 #helpers > section ul li {
   border-bottom: 1px solid #5E6475;
-  padding-bottom: 20px;
-  padding-top: 5px;
+  padding: 5px 5px 20px 5px;
```
Comment (mathjazz) [inline pontoon/base/static/css/style.css]: We don't need to change this.

File: pontoon/base/static/css/search.css [hunk 4]
```diff
@@ -83,3 +83,17 @@
...
 }
+
+.search .clipboard-success {
+  float: left;
+  color: #7BC876;
+}
+
+.search .machinery li {
+  padding: inherit 5px;
+}
```
Comment (mathjazz) [inline pontoon/base/static/css/search.css]: This rule needs to be more strict, otherwise it's ignored.

File: pontoon/base/static/css/search.css [hunk 5]
```diff
@@ -83,3 +83,17 @@
...
+
+.search .machinery li:hover {
+  background-color: #3f4752;
+  cursor: pointer;
+}
```
Comment (mathjazz) [inline pontoon/base/static/css/search.css]: You can remove `.search` from all selectors.

**Comment by mathjazz** [comment]
Added a few nits. Please squash commits. r+

**Comment by mathjazz** [comment]
A thing of beauty!

**Pull request merged**